<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>update</string>
   </property>
   <widget class="QRadioButton" name="radioKOSPI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>코스피</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioKOSDAQ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>코스닥</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="realtimeData">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>실시간데이타</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
